src/app/(proper_react)/redesign/(authenticated)/user/dashboard/fix/data-broker-profiles/manual-remove/RemovalCard.tsx

```diff
@@ -33,6 +35,10 @@ export const RemovalCard = (props: Props) => {
         credentials: "same-origin",
       },
     );
+    // Ensure previously-visited pages that still have this scan result marked
+    // as unfixed are removed from the cache. See
+    // https://nextjs.org/docs/app/building-your-application/caching#invalidation-1
+    router.refresh();
```

Comment:
To copy over [from Slack](https://mozilla.slack.com/archives/C04BLQVN151/p1701947231096259?thread_ts=1701881829.231479&cid=C04BLQVN151):

> I don't think we need it in all network requests, but if we change data on the backend that we also show elsewhere on the website (maybe all/most PUT, POST and PATCH requests?), I suppose so. It also feels like I might still have missed something, because it feels heavy-handed, but then again, this is all pretty new, and they're probably still just figuring out patterns.